tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "strictPropertyInitialization": true,
     "noFallthroughCasesInSwitch": true,
     "resolveJsonModule": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "jsx": "preserve"
```

Comment:
this flag is so we can do react stuff. it was throwing an error and breaking builds